proto/cosmos/tx/v1beta1/service.proto

```diff
@@ -14,9 +15,59 @@ service Service {
         option (google.api.http).post = "/cosmos/tx/v1beta1/simulate";
     }
     // GetTx fetches a tx by hash.
-    rpc GetTx(GetTxRequest) returns (GetTxResponse) {
+    rpc GetTx(GetTxRequest) returns (TxResponse) {
```

Comment:
We should keep `GetTxResponse`, because of buf linting: an rpc method `Abc` should have request called `AbcRequest` and response called `AbcResponse`.

`GetTxResponse` can be just a simple message with one field.